src/qt/forms/optionsdialog.ui

```diff
@@ -252,6 +252,16 @@
             </property>
            </widget>
           </item>
+          <item>
+           <widget class="QCheckBox" name="m_enable_psbt_controls">
+            <property name="text">
+             <string extracomment="An options window setting to enable advanced PSBT controls.">Enable advanced &amp;PSBT controls</string>
```

Comment:
In commit "qt: hide Create Unsigned button behind an expert mode option" (790733e54c3daff547acbc90fb179b48b6d2fb3c)

This is just a personal opinion, but I don't like the "advanced" weasel wording that makes the option more mysterious, implies that if you are an "advanced" user you should already know what this does without any explanation, and implies that if you don't have some outside knowledge what this does, then maybe you are some kind of simpleton.

Easy to fix this by replacing: "Enable advanced PBST controls" with "Enable PSBT controls". Or just literally "Enable create unsigned transaction feature when spending".

Reply:
Removed "advanced"